jmclient/jmclient/wallet-rpc-api.yaml

```diff
@@ -0,0 +1,618 @@
+      properties:
+        walletname:
+          type: string
+          example: wallet.jmdat
+        password:
+          type: string
+          format: password
+          example: hunter2
+        wallettype:
+          type: string
+          example: sw
+    UnlockWalletRequest:
+      type: object
+      required:
+        - password
+      properties:
+        password:
+          type: string
+          format: password
+          example: hunter2
+    DirectSendRequest:
+      type: object
+      required:
+        - mixdepth
+        - amount_sats
+        - destination
+      properties:
+        mixdepth:
+          type: integer
+          example: 0
+        amount_sats:
+          type: integer
+          example: 100000000
+        destination:
+          type: string
+          example: bcrt1qu7k4dppungsqp95nwc7ansqs9m0z95h72j9mze
+    ErrorMessage:
+        type: object
+        properties:
+          message:
+            type: string
+
+  responses:
+    # Success responses
+    DirectSend-202-Accepted:
+      description: "accepted as valid request to send transaction."
+      content:
+        application/json:
+          schema:
+            $ref: "#/components/schemas/DirectSendResponse"
+    ListUtxos-200-OK:
+      description: "successful retrieval of utxo list"
+      content:
+        application/json:
+          schema:
+            $ref: "#/components/schemas/ListUtxosResponse"
+    GetAddress-200-OK:
+      description: "successful retrieval of new address"
+      content:
+        application/json:
+           schema:
+             $ref: "#/components/schemas/GetAddressResponse"
+    ListWallets-200-OK:
+      description: "successful response to listwallets"
+      content:
+        application/json:
+          schema:
+            $ref: "#/components/schemas/ListWalletsResponse"
+    Session-200-OK:
+      description: "successful heartbeat response"
+      content:
+        application/json:
+          schema:
+            $ref: "#/components/schemas/SessionResponse"
+    Create-201-OK:
+      description: "wallet created successfully"
+      content:
+        application/json:
+          schema:
+            $ref: "#/components/schemas/CreateWalletResponse"
+    Unlock-200-OK:
+      description: "wallet unlocked successfully"
+      content:
+        application/json:
+          schema:
+            $ref: "#/components/schemas/UnlockWalletResponse"
+    Display-200-OK:
+      description: "wallet display contents retrieved successfully."
+      content:
+        application/json:
+          schema:
+            $ref: "#/components/schemas/WalletDisplayResponse"
+    Lock-200-OK:
+      description: "wallet locked successfully"
+      content:
+        application/json:
+          schema:
+            $ref: "#/components/schemas/LockWalletResponse"
+    202-Accepted:
+      description: The request has been submittted successfully for processing, but the processing has not been completed. 
```

Comment:
```suggestion
      description: The request has been submitted successfully for processing, but the processing has not been completed. 
```

Reply:
Done in https://github.com/JoinMarket-Org/joinmarket-clientserver/pull/996/commits/938aa7659ff44d7621294e44bb8a244ccf1b00dd